### ISSUE TITLE
Change or remove lowercase `-v` version option

### ISSUE THREAD

**Issue by gerardroche** [php-build/php-build#283]
*I'm waiting on the usage message pr to be merged reviewed before submitting this*

Either:

- change version short option `-v` to upper-case `-V`
- remove it so that only `--version` prints the version

Reason:
    
`-v` is mostly used for verbose logging. Changing this to uppercase or removing it allows the possibility of the verbose modes in a later version without backwards compatability issues e.g. `-v` `-vv` `-vvv`

**Comment by GrahamCampbell** [comment]
`php -v` does the version. Why should we change?

**Comment by gerardroche** [comment]
`-v` is usually verbose mode. `-vv` very verbose, `-vvv` very very verbose. When a system has a verbose mode `-v` is more useful as verbose than it is for version because it will be used more. True, some use it for version, some use a capital v, `-V` as the version short option.

If a verbose mode is added later in php-build it will require a bc break to use able to use `-v`. Maybe a verbose mode will never be introduced. Still, if this is going to change it should change before 1.0.

**Comment by GrahamCampbell** [comment]
> usually verbose mode

Yes, I know, but I was saying we should probably copy what php does.

**Comment by rogeriopradoj** [comment]
@gerardroche, I think we will have to see some projects in each side to decide, ;-)

A small list I've started..

# -v as verbose
- composer
- ruby-build
- vagrant

# -v as version
- php
- rvm
- docker

**Comment by gerardroche** [comment]
Interesting GNU standards: https://www.gnu.org/prep/standards/html_node/Command_002dLine-Interfaces.html

**Comment by henriquemoody** [comment]
As I see, using `-V` for version makes more sense when you actually have verbose mode. Does php-build intent to have this feature? Because if not (as php, rvm and docker haven't) it doesn't makes much sense, IMHO.

Also, it's a big BC break, and if php-build will implement some feature like that in the feature it can use `--debug`, like rvm does.

**Comment by GrahamCampbell** [comment]
TBH, why do we even need a version command anyway? We haven't tagged a release in way over a year?

**Comment by madumlao** [comment]
Necrobump.

I might need to implement `-v` verbose mode soon for consistency across different *env environments. Currently maintaining [phpenv](https://github.com/madumlao/phpenv) but also using pyenv, nenv, rbenv, etc... Please consider the PR when it comes.

So to add to the list:
- rbenv/ruby-build: uses -v verbose (sorta kinda upstream?)
- pyenv: uses -v verbose
- nenv: uses -v verbose
- goenv: uses -v verbose

Seems to me like more *env-style environments uses -v verbose than not (there are some that don't, crenv/crystal-build). And yes, it's so much fun having half a dozen dynamic languages each with their own quirks that don't like system packagers in production ;)
